recipes/icu/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+if [ "$(uname)" == "Linux" ]; then
+    ./configure --prefix="$PREFIX"
```

Comment:
Should we have some of the same options from the Mac/Darwin section (e.g. `--disable-samples`, `--enable-static`) up here? Also, should we be configuring whether it is 32-bit or 64-bit?